CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ option(S2N_INTERN_LIBCRYPTO "This ensures that s2n-tls is compiled and deployed
 version of libcrypto by interning the code and hiding symbols. This also enables s2n-tls to be
 loaded in an application with an otherwise conflicting libcrypto version." OFF)
 option(S2N_LTO, "Enables link time optimizations when building s2n-tls." OFF)
+option(S2N_NO_STACKTRACE "Disables stacktrace functionality in s2n-tls." OFF)
```

Comment:
having "no" in feature names makes things more difficult to follow. I would rather have the default be `ON`

```suggestion
option(S2N_STACKTRACE "Enables stacktrace functionality in s2n-tls." ON)
```

Reply:
That makes sense! Fixed in f620d574032bd9f1e4b073670e2c273870f1baa8